### PR TITLE
Blog - Apply Mutual TLS Over a Kubernetes Nginx Ingress Controller

### PR DESCRIPTION
**Pull request by msali123**
Done with all checks.

### CODE REVIEW

**Comment by msali123** [comment]
@jalletto 
Preview URL:
https://63ed26578e402500658432db--ephemeral-narwhal-3ac3b2.netlify.app

File: blog/_posts/2023-03-28-mutual-tls-kubernetes-nginx-ingress-controller.md [hunk 1]
```diff
@@ -0,0 +1,307 @@
...
+Now we want to hit the hostname (test.localdev.me) on port 443 to test the HTTPS connection through the port-forward technique. The below command will forward traffic from port 443 on the local machine to the Nginx Ingress Controller service running on port 443 inside the namespace `ingress-nginx`.
+
+~~~{.bash caption=">_"}
+$ sudo kubectl port-forward -n ingress-nginx \
+service/ingress-nginx-controller 443:443
+~~~
+
+**Note**: `sudo` is required here to allow incoming traffic on port 443.
+Also keep that command running in a separate session as port-forwarding will be used for the rest of the article.
+
```
Comment (balapriyac) [inline blog/_posts/2023-03-28-mutual-tls-kubernetes-nginx-ingress-controller.md:191]:
```suggestion

<div class="notice--info">
  `sudo` is required here to allow incoming traffic on port 443.
Also keep that command running in a separate session as port-forwarding will be used for the rest of the article.
</div>
```

File: blog/_posts/2023-03-28-mutual-tls-kubernetes-nginx-ingress-controller.md [hunk 2]
```diff
@@ -0,0 +1,307 @@
...
+From another terminal, let's try to curl the local endpoint with HTTPS protocol:
+
+~~~{.bash caption=">_"}
+$ curl -k -v https://test.localdev.me/
+~~~
+
+ -k is used to skip self-signed certificate verification and -v to see some logs.
```
Comment (balapriyac) [inline blog/_posts/2023-03-28-mutual-tls-kubernetes-nginx-ingress-controller.md:198]:
```suggestion
 `-k` is used to skip self-signed certificate verification and `-v` to see some logs.
```

**Review by balapriyac — CHANGES_REQUESTED** [review]
Thanks Sameer. @jalletto I've finished reviewing this PR.

**Comment by msali123** [comment]
Preview URL:
https://63f3328c565a8320aaa0981a--ephemeral-narwhal-3ac3b2.netlify.app

@balapriyac @jalletto changes done

**Review by jalletto — APPROVED** [review]

**Pull request merged**